as.c

```diff
@@ -3,6 +3,7 @@
 #include <string.h>
 
 #include "vm.h"
+#include "opcode.h"
```

Comment:
You can generate `opcode.h` by looking up given opcode description file with customized script.